snuba/stateful_consumer/states/bootstrap.py

```diff
@@ -38,10 +114,32 @@ def __init__(self,
             on_message=self.__handle_msg,
         )
 
+        self.__recovery_state = RecoveryState()
+
     def __handle_msg(self, message: Message) -> CommitDecision:
-        # TODO: Actually do something with the messages and drive the
-        # state machine to the next state.
-        return CommitDecision.DO_NOT_COMMIT
+        value = json.loads(message.value())
+        parsed_message = parse_control_message(value)
+
+        current_snapshot = self.__recovery_state.get_active_snapshot_msg()
```

Comment:
This assignment doesn't have anything to do with the code immediately following it, it should be moved down closer to the usage unless there's some reason it needs to be here (if there is, it should be commented.)

Reply:
The lines below willchange the result of get_active_snapshot_msg, so this has to be before. It will be compared with new_snap later